### PR TITLE
Feature public feed

### PR DESCRIPTION
**Pull request by mcardosog**
Feature Public Feed updated.
Added HTML (feed.html) page.
Added JS file containing code that loads all the messages from the user.

### CODE REVIEW

File: src/main/webapp/user-page.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <meta charset="UTF-8">
     <link rel="stylesheet" href="/css/main.css">
     <link rel="stylesheet" href="/css/user-page.css">
-    <script src="/js/user-page-loader.js"></script>
+    git c
```
Comment (ctrokid) [inline src/main/webapp/user-page.html]: Do we need this?

**Review by ctrokid — CHANGES_REQUESTED** [review]

File: src/main/webapp/user-page.html (hunk 1 above)
Comment (ctrokid) [reply src/main/webapp/user-page.html]: I also don't know if we want to be removing the import of user-page-loader.js, do we?

Comment (mcardosog) [reply src/main/webapp/user-page.html]: Apparently when I was trying to commit I had that line selected and I typed "git c" in the IDE instead the in the terminal 😭.
On the other hand I didn't change anything on that file. I believe that line 24 "<script src="/js/user-page-loader.js"></script>" is necessary ???

**Review by mcardosog — COMMENTED** [review]
Added "<script src="/js/user-page-loader.js"></script>" to user-page.html

**Review by mcardosog — COMMENTED** [review]
Accidentally removed line containing "<script src="/js/user-page-loader.js"></script>".
Line added (24)

File: src/main/webapp/user-page.html (hunk 1 above)
Comment (ctrokid) [reply src/main/webapp/user-page.html]: Lol I hate when that happens, I have definitely done this before.

Yepp, it's necessary. Thanks for adding it back.

**Review by ctrokid — APPROVED** [review]

**Pull request merged**